docs/modules/community/pages/mailing-lists.adoc

```diff
@@ -0,0 +1,165 @@
+= Mailing lists
+
+== Guidelines
+
+A mailing list is an electronic discussion forum that anyone can subscribe to. When someone sends an email message to
+the mailing list, a copy of that message is broadcasted to everyone who is subscribed to that mailing list.
+Mailing lists provide a simple and effective communication mechanism for discussion and decision making.
+
+The *Apache Software Foundation* has well established reasons for using email and not other types of forum.
+We are *not* interested in hearing proposals to use NNTP (news groups) or web forums or any other medium.
```

Comment:
I think we can remove that aggressive "NNTP" statement. (L 10-11) it never happened in years.